credentialMixin.js

```diff
@@ -102,7 +102,7 @@ export const credentialMixin = {
 };
 
 function _createFields(fields, source, schema) {
-  for(const key in source) {
+  for(const key in schema) {
```

Comment:
This ensures that fields are rendered in schema order, not object order, which was wrong.